docs/reference/limitations/object-storage.md

```diff
@@ -34,3 +34,49 @@ We do not support `GOVERNANCE` mode.
 ### SSE-KMS
 
 There is currently no support for SSE-KMS in {{brand}}. SSE-C, in contrast, [is fully supported](../../howto/object-storage/s3/sse-c.md).
+
+### Request checksum calculation
+
+Due to recent changes in AWS SDKs in early 2025, uploading objects to {{brand}} object storage services may fail when using tools that depend on those SDKs.
+Calls using the previously optional checksum requirement will fail because our backend storage doesn't recognize these requests.
+
+This can be resolved by setting the environment variable `AWS_REQUEST_CHECKSUM_CALCULATION` to `WHEN_REQUIRED` or by changing the AWS CLI defaults.
+Other tools can have similar options that can be changed to retain compatibility.
+
+=== "aws"
+    ```sh
+    export AWS_REQUEST_CHECKSUM_CALCULATION=WHEN_REQUIRED
+    ```
+
+    or
+
+    ```ini
+    [default]
+    request_checksum_calculation = WHEN_REQUIRED
+    ```
+=== "Terraform/OpenTofu"
```

Comment:
Weird, my earlier comment didn't register, so here it is again: the "for better compatibility" part belongs in the `aws` tab. Like so:

```suggestion

  For better compatibility, ensure you're using a current version (>=2.23.5) of the AWS CLI that should accept `AWS_REQUEST_CHECKSUM_CALCULATION` environment variable and config options.
=== "Terraform/OpenTofu"
```